# 演習問題19

In [1]:
from scipy import stats

In [4]:
eater = [1.2, 0.8, 1.5, 1.2, 1.0]
not_eater = [0.8, 0.4, 0.6, 1.0, 0.5, 0.4, 0.5]

In [5]:
stats.ttest_ind(eater, not_eater)

Ttest_indResult(statistic=3.8560188172363903, pvalue=0.0031808361733834186)

検定統計量Tの実現値t=3.856は棄却域R(pvalue<0.05)に入る。
よって、仮説H0:μx=μyは棄却される